contact-me.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact Me</title>
+</head>
+<body>
+    <section>
+        <header>
+            <section>
+                <figure> 
+                    <img src="../ss22-carolllemieux/images/logocl.PNG" alt="Logo overlapping CL Carol LeMieux's initials" style="width:200px;height:200px;">
```

Comment:
Change `src="../ss22-carolllemieux/images/logocl.PNG"` to `src="images/logocl.PNG"`